src/Parser.ts

```diff
@@ -701,7 +701,7 @@ export const Monad: Monad2<URI> = {
 
 /**
  * @category instances
- * @since 0.6.11
+ * @since 0.6.12
```

Comment:
I think we can keep 0.6.11, only the implementation has changed

Reply:
Makes sense! Done 👍